satpy/scene.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# Copyright (c) 2010-2017 Satpy developers
+# Copyright (c) 2010-2017, 2022 Satpy developers
```

Comment:
Why not add to the range? `2010-2022`? We could also discuss just removing the years...or the copyright and license header altogether (as I think dask and other projects have started doing).

Reply:
Yes, this is just handled automatically in my IDE (emacs :-) )
I let you decide. Now I have added the full range. Think it requires a more over reaching discussion and decision if we should remove the headers alltogether